tests/sources/tools/utils/test_hyperparameter_tuner.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from optuna.study.study import Study
```

Comment:
I think this is generally ok. One concern that I have is the mixing of optuna imports with OpenDR imports. This is not something that needs to be changed, however I was wondering if we should expose optuna through OpenDR, e.g., changing

```python
from optuna.study.study import Study
````
to (which can include two imports into one)
```python
from opendr.utils.hyperparameter_tuner import HyperparameterTuner, Study
````

However, feel free to dismiss this comment, if you think that this will pollute the interface.

Reply:
I see your point, but I don't think this will improve the usability. The common way to create a study is using the optuna.create_study() method so in practice a user is not using the Study object. Thanks however for the suggestion!